### PR TITLE
make it Fedora compatible

### PR DESCRIPTION
**Pull request by emmawu**
- enabled sysrq
- install kernel-debuginfo from federo-debuginfo repo
- fixed an error in func install_rpm_from_repo()
- removed some empty lines and comments

Signed-off-by: xiawu <xiawu@redhat.com>

### CODE REVIEW

File: dump-ssh-ipv4/runtest.sh [hunk 1]
```diff
@@ -18,11 +18,9 @@
 # Author: Song Qihan<qsong@redhat.com>
 # Update: Qiao Zhao <qzhao@redhat.com>
 
-# Source necessary library
 . ../lib/kdump.sh
 . ../lib/kdump_report
```
Comment (lulinqing) [inline dump-ssh-ipv4/runtest.sh]: missed extension for kdump_report(.sh)?

File: lib/kdump.sh [hunk 2]
```diff
@@ -81,10 +100,10 @@ install_rpm()
 # @param2: repo
 install_rpm_from_repo()
 {
-    if [[ $# -gt 2 ]]; then
+    if [[ $# -gt 1 ]]; then
         local pkg=$1
         local repo=$2
-        rpm -q $pkg || yum install -y --enablerepo=$1 $2
+        rpm -q $pkg || yum install -y --enablerepo=$repo  $pkg
```
Comment (lulinqing) [inline lib/kdump.sh]: extra blank between two var names

File: lib/kdump.sh [hunk 3]
```diff
@@ -81,10 +100,10 @@ install_rpm()
 # @param2: repo
 install_rpm_from_repo()
 {
-    if [[ $# -gt 2 ]]; then
+    if [[ $# -gt 1 ]]; then
```
Comment (lulinqing) [inline lib/kdump.sh]: i'd suggest to use "-ge 2"

File: dump-ssh-ipv4/runtest.sh (hunk 1 above)
Comment (emmawu) [inline dump-ssh-ipv4/runtest.sh]: Thanks, fixed.

File: lib/kdump.sh (hunk 3 above)
Comment (emmawu) [inline lib/kdump.sh]: Fixed.

File: lib/kdump.sh (hunk 2 above)
Comment (emmawu) [inline lib/kdump.sh]: Fixed.

File: lib/crash.sh [hunk 4]
```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env bash
 
-((LIB_CRASH_SH)) && return || LIB_CRASH_SH=1
+# Basic Library for Kdump Multi-Host Test
+
+# Copyright (C) 2016 Song Qihan <qsong@redhat.com>
```
Comment (sztsian) [inline lib/crash.sh]: You used a not proper GPL v3 license statement. 
A better one is https://github.com/kdump-test/kdump-test/blob/master/clean-up/runtest.sh

File: lib/log.sh [hunk 5]
```diff
@@ -15,7 +15,8 @@
 #
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
+#
+# Author: Ziqian Sun <zsun@redhat.com>
```
Comment (sztsian) [inline lib/log.sh]: And why I am here? I am not the author for this file. Please see git log. Change it to qsong or qzhao instead.

File: lib/crash.sh (hunk 4 above)
Comment (sztsian) [inline lib/crash.sh]: This is not a proper statement for the license. See https://github.com/kdump-test/kdump-test/blob/master/clean-up/runtest.sh for a better one IMO.

File: lib/kdump.sh [hunk 6]
```diff
@@ -1,5 +1,24 @@
 #!/usr/bin/env bash
 
+# Basic Library for Kdump Multi-Host Test
+
+# Copyright (C) 2016 Song Qihan <qsong@redhat.com>
```
Comment (sztsian) [inline lib/kdump.sh]: This as well

File: lib/metadata [hunk 7]
```diff
@@ -3,4 +3,5 @@ name=/kdump-test/lib
 own="Qiao Zhao"
 description="kdump-test lib"
 license="GPLv2"
```
Comment (sztsian) [inline lib/metadata]: Please, can you change this to GPLv3 by the same time? Thanks.

**Comment by emmawu** [comment]
Fixed all licenses and other errors.

File: lib/crash.sh [hunk 8]
```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env bash
 
-((LIB_CRASH_SH)) && return || LIB_CRASH_SH=1
+# Library for Crash Test
+
+# Copyright (C) 2016 Song Qihan <qsong@redhat.com>
```
Comment (sztsian) [inline lib/crash.sh]: I'm sorry but here still needs modification.

Comment (emmawu) [inline lib/crash.sh]: Fixed.

**Pull request merged**